frontend/src/components/admin-page/edit-user.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import clsx from 'clsx';
+import { EditUserPayloadKey, UserSex, UserType } from 'common/enums';
+import styles from './styles.module.scss';
+import { IEditUserPayload, IUser } from 'common/interfaces';
+import { validationEditUserSchema } from 'helpers';
```

Comment:
```suggestion
import { validationEditUser as validationEditUserSchema } from 'validation-schemas';
```
It isn't helper
Default name shouldn't contain the `schema`